internal/web/controllers/users-controller.go

```diff
@@ -45,6 +45,22 @@ func (c *UsersController) NewUser(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusCreated)
 }
 
+func (c *UsersController) GetUser(w http.ResponseWriter, r *http.Request) {
```

Comment:
Mesma questão de nomenclatura do service.
```suggestion
func (c *UsersController) GetUserById(w http.ResponseWriter, r *http.Request) {
```